projects/avodata/firstStage.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import math
+
+
+def factorial(n):
+	return int(math.gamma(n + 1))
+
+
+def n_choose_k(n, k):
+	return factorial(n) / (factorial(k) * factorial(n - k))
+
+
+def find_c_and_a(m):
```

Comment:
`find_c_and_a` is not what this method does.
I think it's a little confusing that we're combining the algorithm and the generation of synthetic data to check it.

I propose the following sketch:
1. `generate_synthetic_A` which receives `num_of_professions` and returns `A`
2. `generate_synthetic_c` which receives `m` and `i` and returns `c`.
They are called synthetic because normally they would be the result of user forms.
2. `solve` which receives `A` and `c` and returns a vector of `m`
3. `run_check` which receives `m`, for each vector generates synthetic data (you actually need to create `A` only once since it only depends on the structure of `m`), and then calls `solve`.

Reply:
For the first three points I think I understood and changed exactly as you wanted. For the last point, I created a new matrix M and a matrix of the same size with the errors, that's what you mean?

Reply:
As the output of `run_check`? Perfect.